ipaserver/install/server/__init__.py

```diff
@@ -335,21 +335,29 @@ def dirsrv_config_file(self, value):
     def __init__(self, **kwargs):
         super(ServerInstallInterface, self).__init__(**kwargs)
 
+        if self.domain_level == constants.DOMAIN_LEVEL_0:
+            if (self.no_pkinit or self.pkinit_cert_files is not None or
+                    self.pkinit_pin is not None):
+                raise RuntimeError(
+                    "pkinit on domain level 0 is not supported. Please don't "
```

Comment:
Sure, I will put the behavior of `no-pkinit` on domain level 0 to the correct man pages.
The reasoning to why this certain change was done is to be found in the commit message.